package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "bluebird": "^3.1.5",
     "flow-bin": "^0.21.0",
+    "shelljs": "^0.6.0",
```

Comment:
Using [shelljs](https://github.com/shelljs/shelljs) for cross platform support but mostly for really sane scripting in node.